spec/system/authz_admin_spec.rb

```diff
@@ -26,7 +26,7 @@
         login_as research_data_moderator
         visit edit_work_path(work)
         expect(page).to have_content("Editing Dataset")
-        fill_in "title_main", with: "New Title", fill_options: { clear: :backspace }
+        fill_in "title_main", with: "New Title", fill_options: { clear: :true }
```

Comment:
This seems to be needed now that we are using `selenium` instead of `selenium-chrome' (see https://github.com/pulibrary/pdc_describe/pull/1441)